pkg/cmd/root.go

```diff
@@ -129,6 +130,7 @@ func init() {
 
 	RootCmd.PersistentFlags().BoolVarP(&cachevar, "no-cache", "c", false, "no caching")
 	RootCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", "yaml", "output format yaml or json")
+	RootCmd.PersistentFlags().BoolVarP(&debugSwitch, "debug", "d", false, "enable debug level output")
```

Comment:
```suggestion
	RootCmd.PersistentFlags().BoolVarP(&debugSwitch, "debug", "d", false, "enable debug level output")
```

Could we name it "verbose" as it is not necessarily for debugging? 
Just a personal preference though.

Reply:
Mhm I see that there is already a previous commit that introduces the `debug` flag.
Maybe @DockToFuture has an opinion about the naming of the flag aswell?

Reply:
sure

Reply:
So you want to have the flag named `--debug`  and not `--verbose` ?

Reply:
I mean after rebase it change back. I am fine with `verbose` I can change `--debug` to `--verbose`

Reply:
done. @danielfoehrKn  let me know if there anything else need update.